website/docs/language/values/variables.mdx

```diff
@@ -191,21 +191,6 @@ variable "session_token" {
   type      = string
   ephemeral = true
 }
-
-data "aws_secretsmanager_secret_version" "db_master" {
-  secret_id = data.aws_db_instance.example.master_user_secret[0].secret_arn
-}
-locals {
-  credentials = jsondecode(data.aws_secretsmanager_secret_version.db_master.secret_string)
-}
-
-provider "postgresql" {
-  host     = data.aws_db_instance.example.address
-  port     = data.aws_db_instance.example.port
-  username = local.credentials["username"]
-  password = local.credentials["password"]
-}
-
```

Comment:
I made it more confusing in my gist as I didn't realise I shared the version with the data source - apologies!

This was supposed to be something like this:

```hcl
ephemeral "aws_secretsmanager_secret_version" "db_master" {
  secret_id = data.aws_db_instance.example.master_user_secret[0].secret_arn
}
locals {
  credentials = jsondecode(ephemeral.aws_secretsmanager_secret_version.db_master.secret_string)
}

provider "postgresql" {
  host     = data.aws_db_instance.example.address
  port     = data.aws_db_instance.example.port
  username = local.credentials["username"]
  password = local.credentials["password"]
}
```

I'm fine with keeping it small for now though.

Reply:
Oooh I love this example! 

@robin-norwood Could you please also add this to that new ephemeral resources page?